addons/binding/org.openhab.binding.network/ESH-INF/i18n/network_de.properties

```diff
@@ -1,46 +1,46 @@
 thing-type.config.network.pingdevice.hostname.label = Hostname oder IP
-thing-type.config.network.pingdevice.hostname.description = Hostname oder IP des Netzwerkgerätes
+thing-type.config.network.pingdevice.hostname.description = Hostname oder IP des Netzwerkger�tes
 thing-type.config.network.pingdevice.retry.label = Wiederholen
-thing-type.config.network.pingdevice.retry.description = Gibt an, wie oft der PING wiederholt werden soll, bevor das Gerät als offline markiert wird.
+thing-type.config.network.pingdevice.retry.description = Gibt an, wie oft der PING wiederholt werden soll, bevor das Ger�t als offline markiert wird.
 thing-type.config.network.pingdevice.timeout.label = Zeitlimit
-thing-type.config.network.pingdevice.timeout.description = Gibt an, wie lange maximal gewartet wird (in ms), bevor ein Gerät als nicht vorhanden gekennzeichnet wird.
+thing-type.config.network.pingdevice.timeout.description = Gibt an, wie lange maximal gewartet wird (in ms), bevor ein Ger�t als nicht vorhanden gekennzeichnet wird.
 thing-type.config.network.pingdevice.refresh_interval.label = Aktualisierungsintervall
 thing-type.config.network.pingdevice.refresh_interval.description = Spezifiziert den Aktualisierungsintervall (in ms)
 
-thing-type.network.servicedevice.label = Netzwerkgerät mit Dienst
-thing-type.network.servicedevice.description = Die Verfügbarkeit des Geräts wird durch einen Verbindungsversuch mit dem angegeben TCP Dienst festgestellt.
+thing-type.network.servicedevice.label = Netzwerkger�t mit Dienst
+thing-type.network.servicedevice.description = Die Verf�gbarkeit des Ger�ts wird durch einen Verbindungsversuch mit dem angegeben TCP Dienst festgestellt.
 thing-type.config.network.servicedevice.hostname.label = Hostname oder IP
-thing-type.config.network.servicedevice.hostname.description = Hostname oder IP des Netzwerkgerätes
+thing-type.config.network.servicedevice.hostname.description = Hostname oder IP des Netzwerkger�tes
 thing-type.config.network.servicedevice.retry.label = Wiederholen
-thing-type.config.network.servicedevice.retry.description = Gibt an, wie oft der Verbindungsversuch wiederholt werden soll, bevor das Gerät als offline markiert wird.
+thing-type.config.network.servicedevice.retry.description = Gibt an, wie oft der Verbindungsversuch wiederholt werden soll, bevor das Ger�t als offline markiert wird.
 thing-type.config.network.servicedevice.timeout.label = Zeitlimit
-thing-type.config.network.servicedevice.timeout.description = Gibt an, wie lange maximal gewartet wird (in ms), bevor ein Gerät als nicht vorhanden gekennzeichnet wird.
+thing-type.config.network.servicedevice.timeout.description = Gibt an, wie lange maximal gewartet wird (in ms), bevor ein Ger�t als nicht vorhanden gekennzeichnet wird.
 thing-type.config.network.servicedevice.refresh_interval.label = Aktualisierungsintervall
 thing-type.config.network.servicedevice.refresh_interval.description = Spezifiziert den Aktualisierungsintervall (in ms)
```

Comment:
Should be "das Aktualisierungsintervall"

Reply:
@claell I will probably not be available for some days in some days, would be awesome if we don't block this PR by i18n. This is not part of this PR, I just correct the encoding problem here.

Reply:
No problem, this was probably not the right place to note this. I will open an issue to keep track of that then.